website/docs/gcp_bigquery.md

```diff
@@ -5,7 +5,24 @@ summary: Introduce BigQuery integration in Hudi.
 ---
 
 Hudi tables can be queried from [Google Cloud BigQuery](https://cloud.google.com/bigquery) as external tables. As of
-now, the Hudi-BigQuery integration only works for hive-style partitioned Copy-On-Write tables.
+now, the Hudi-BigQuery integration only works for hive-style partitioned Copy-On-Write and Read-Optimized Merge-On-Read tables.
+
+## Sync Modes
+### Manifest File
+As of version 0.14.0, the `BigQuerySyncTool` supports syncing table to BigQuery using [manifests](https://cloud.google.com/blog/products/data-analytics/bigquery-manifest-file-support-for-open-table-format-queries). On the first run, the tool will create a manifest file representing the current base files in the table and a table in BigQuery based on the provided configurations. The tool produces a new manifest file on each subsequent run and will update the schema of the table in BigQuery if the schema changes in your Hudi table.
+#### Benefits of using the new manifest approach:
+<ol>
+	<li>Only the files in the manifest can be scanned leading to less cost and better performance for your queries</li>
+	<li>The schema is now synced from the Hudi commit metadata allowing for proper schema evolution</li>
+	<li>Lists no longer have unnecessary nesting when querying in BigQuery as list inference is enabled by default</li>
+	<li>Partition column no longer needs to be dropped from the files due to new schema handling improvements</li>
+</ol>
+
+### View Over Files (Legacy)
+After run, the sync tool will create 2 tables and 1 view in the target dataset in BigQuery. The tables and the view
```

Comment:
Copied this from below

Reply:
can you add a line, this is what was supposed pre 0.14.0. and may be out of the box behavior w/ 0.14.0 (depending on if manifest file based is enabled by default or not)